python/pdo/submitter/ccf/ccf_submitter.py

```diff
@@ -115,7 +114,7 @@ def __init__(self, ledger_config, *args, **kwargs):
         try:
             parsed_url = urlparse(self.url)
             host, port = parsed_url.netloc.split(':')
-            self.host = socket.gethostbyname(host) # convert host name to IP address
+            self.host = host
             self.port = int(port)
```

Comment:
again... this needs to work with both dns names and addresses. it seems pretty reasonable to convert everything we can to IP address.

Reply:
If it's necessary, I understand. Otherwise, why would we do that? It's even harder to debug proxy issues.

Reply:
get your no_proxy configured with local addresses (including 172.0.0.0/8) appropriately & see if it works. the requests client that the ccf submitter uses can handle network specifications in no_proxy.